packages/astro/import-meta.d.ts

```diff
@@ -1,4 +1,4 @@
-// File vendored from Vite itself, as a workaround to https://github.com/vitejs/vite/pull/9827 until Vite 4 comes out
+// File vendored from Vite itself, as a workaround to https://github.com/vitejs/vite/pull/9827 until Vite 3.2 comes out
```

Comment:
There's still issues in this file, because Vite doesn't have its types files exported so we can't refer to them. 

Vite is changing how its types work in 3.2 (next release), so I think we can tolerate this for now as the effects are pretty minor (you just don't get intellisense on import.meta, but no errors)

Reply:
I've changed the types export for Vite 3.2 so I'm not sure if it will fix this vendoring part. We might have to wait till Vite 4 to have `import-meta` be lowercase.

I had to change it before as Intellisense is showing `ImportMeta_2` instead of just `ImportMeta` due to bundling. It felt a bit confusing for the users otherwise.

Reply:
Ah, that's unfortunate. I'm not sure how we can fix this on our side at the moment (outside of vendoring all the types) then. It'd be helpful if Vite exported its types through the `exports` field

Reply:
Yeah in Vite 4 we can switch to using `import-meta.d.ts` to work around the problem as previously planned 😅 Maybe  we should revert this line change for now.